### PR TITLE
Add right click functionalities

### PR DESCRIPTION
**Pull request by kemoycampbell**
This feature add the ability to perform right click functionality such as:

- Search with Google,
- Copy Image
- Copy Image Address
- Save Link
- Save Image as
- Show inspect
- Look up selection

This is done by using the electron-context-menu package.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5967 @@
...
+    "ejs": {
+      "version": "3.1.6",
+      "resolved": "https://registry.npmjs.org/ejs/-/ejs-3.1.6.tgz",
+      "integrity": "sha512-9lt9Zse4hPucPkoP7FHDF0LQAlGyF9JVpnClFLFH3aSSbxmyoqINRpp/9wePWJTUl4KOQwRL72Iw3InHPDkoGw==",
+      "dev": true,
+      "requires": {
+        "jake": "^10.6.1"
+      }
+    },
+    "electron": {
```
[the remark this answers is not in the document]
Comment (Dev727) [reply package-lock.json:4308]: Added this to my personal fork, and it seems to work great!

Hopefully we can see an update to the original fork...

**Comment by Dev727** [comment]
Added this to my own persona fork and it works great!

Hopefully we can see an update to original...

**Comment by mamaraddio** [comment]
> Added this to my own persona fork and it works great!
> 
> Hopefully we can see an update to original...

@julian-alarcon

**Comment by julian-alarcon** [comment]
Hello! Thank for your contribution @kemoycampbell .
For now I will like to keep lean the client and avoid adding more external dependencies, also we are already supporting right click to view the url of the links, maybe we could add something for the specific more needed/wanted actions.

**Pull request closed without merging**